### PR TITLE
Add job list for localization id_ID

### LINKED ISSUE #337: jobTitle() returns random value for localization id_ID
### Summary

jobTitle() is not returning expected value, because the list of jobs are not available for localization id_ID. I already make pull request for fixing this issue.

### Versions

|                | Version |
|:---------------|:--------|
| PHP            | 8.0.8   |
| `fakerphp/faker`  | 1.14.0   |

### Self-enclosed code snippet for reproduction

```php
require 'vendor/autoload.php';
$faker = Faker\Factory::create('id_ID');
echo $faker->jobTitle();
```

### Expected output

```txt
# return job titles in Indonesian
```

### Actual output

```txt
# return random value of string
```

### PR DESCRIPTION
**Pull request by CikiMomogi**
### What is the reason for this PR?

- [ ] A new feature
- [x] Fixed an issue (resolve #337)

### Author's checklist

- [ ] Follow the [Contribution Guide](https://github.com/FakerPHP/Faker/blob/main/.github/CONTRIBUTING.md)
- [ ] New features and changes are [documented](https://github.com/FakerPHP/fakerphp.github.io)

### Review checklist

- [ ] All checks have passed
- [ ] Changes are approved by maintainer

### CODE REVIEW

File: src/Faker/Provider/id_ID/Company.php [hunk 1]
```diff
@@ -15,6 +15,15 @@ class Company extends \Faker\Provider\Company
      * @see http://id.wikipedia.org/wiki/Jenis_badan_usaha
      */
     protected static $companyPrefix = ['PT', 'CV', 'UD', 'PD', 'Perum'];
+    
+
```
Comment (pimjansen) [inline src/Faker/Provider/id_ID/Company.php:19]: Remove extra newline

**Review by pimjansen — CHANGES_REQUESTED** [review]
Build failing

File: src/Faker/Provider/id_ID/Company.php (hunk 1 above)
Comment (CikiMomogi) [reply src/Faker/Provider/id_ID/Company.php]: Extra newline removed

Comment (pimjansen) [reply src/Faker/Provider/id_ID/Company.php]: > Extra newline removed ([fb28453](https://github.com/FakerPHP/Faker/pull/338/commits/fb28453718417e6dcf3582cfdf14b07bf3c517c3))

Build still failing. Please review the codestyle

**Comment by CikiMomogi** [comment]
Cancel this pull request. Create the new one with already tested build

**Comment by pimjansen** [comment]
> Cancel this pull request. Create the new one with already tested build

This is pretty useless? You can just update the existing PR without an issue?

**Pull request closed without merging**